.github/workflows/irs-build.yml

```
@@ -110,7 +110,7 @@ jobs:
       - uses: actions/checkout@v4
 
       # Needed to create multi-platform image
-      - name: Set up Docker Buildx
+      - name: Set up Docker Build
```

```suggestion
      - name: Set up Docker Buildx
```
"Buildx" is the correct name for the docker BuildKit capabilities